packages/core/src/connectivity/scp-cf/token-accessor.ts

```diff
@@ -24,19 +24,20 @@ import {
 
 async function getClientCredentialsToken(
   service: string | Service,
-  userJwt?: string
+  userJwt?: string | JwtPayload
 ): Promise<ClientCredentialsResponse> {
   const serviceCredentials = resolveService(service).credentials;
 
   let subdomain: string;
   let zoneId: string;
 
   if (userJwt) {
-    const decodedJwt = decodeJwt(userJwt);
-    if (decodedJwt.iss) {
-      subdomain = parseSubdomain(decodedJwt.iss);
+    const jwtPayload =
+      typeof userJwt === 'string' ? decodeJwt(userJwt) : userJwt;
+    if (jwtPayload.iss) {
```

Comment:
Is this working for IAS token where the iss subdomain is not the one from the subaccoutn?

Reply:
From the IAS token scenario, the `iss` key has the value, which is the default IdP demain: `https://XX.accountsXXX.ondemand.com`

Reply:
As discussed with Junjie, this will become relevant once he works on IAS.